src/resources/ble_esp32.cc

```diff
@@ -1889,7 +1889,7 @@ PRIMITIVE(get_value) {
 }
 
 PRIMITIVE(write_value) {
-  ARGS(BleReadWriteElement, element, Object, value, bool, with_response)
+  ARGS(BleReadWriteElement, element, Object, value, bool, with_response, bool, flush)
```

Comment:
This feels weird. Why do you take two bool arguments and only act on the logical or of them?